etl/queries.py

```diff
@@ -60,9 +61,9 @@
        w.ROW_EXPDATE,
        l.NUMVALUE AS CTM_SEGMENT_ID
 FROM MAXIMO_DM.FCT_WORKORDER w
-         LEFT JOIN MAXIMO_DM.DIM_LOCATIONSPEC l ON w.LOCATION = l.LOCATION
+         LEFT OUTER JOIN (SELECT * from MAXIMO_DM.DIM_LOCATIONSPEC where ASSETATTRID = 'GIS_SEGMENT_ID') l
```

Comment:
cool - i see that this combines the old join and filter that used to be on line 67 👍